docs/src/examples/redis-clustering.md

```diff
@@ -2,72 +2,52 @@
 
 The following guide shows you how to configure Shotover Proxy to support transparently proxying Redis cluster _unaware_ clients to a [Redis cluster](https://redis.io/topics/cluster-spec).
 
-## Overview
+## Setting up the redis cluster
 
-In this example, we will be connecting to a Redis cluster that has the following topology:
+First you need to setup a redis cluster for shotover to connect to.
 
-* `192.168.0.1:6379`
-* `192.168.0.2:6379`
-* `192.168.0.3:6379`
-* `192.168.0.4:6379`
-* `192.168.0.5:6379`
-* `192.168.0.6:6379`
+The easiest way to do this is with this example [docker-compose.yml](https://github.com/shotover/shotover-proxy/blob/main/shotover-proxy/examples/redis-cluster/docker-compose.yml)
+You should first inspect the docker-compose.yml to understand what the cluster looks like and how its exposed to the network.
 
-Shotover will listen on the loopback adapter (localhost) and act as a sidecar for our application that speaks Redis. In this example we will use `redis-benchmark` as our Redis cluster unaware client application.
+Then run:
 
-## Configuration
+```bash
+curl -L https://raw.githubusercontent.com/shotover/shotover-proxy/main/shotover-proxy/examples/redis-cluster/docker-compose.yml --output docker-compose.yml
+docker-compose -f docker-compose.yml up
+```
+
+When you are finished with the containers ctrl-c will shut them down.
```

Comment:
`<kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>Space</kbd>` = <kbd>Ctrl</kbd> + <kbd>C</kbd>

Reply:
Oh that's cool!

Reply:
![image](https://user-images.githubusercontent.com/5120858/140031190-d4d10b0b-7efe-4423-be2d-35d953d81c0b.png)
doesnt seem to work :cry: